cpp/README.md

```diff
@@ -43,12 +43,11 @@ If the compilation is successful, `*.tvc` file is created in the directory where
 
 ## Contract deployment
 
-Here we describe a contract deployment to the TON Blockchain Test Network (testnet) using `tonlabs-cli` tool.
-Note: for MacOS X replace `tonlabs-cli` with `cargo run` in the following instruction, and the run is supposed to be made from the directory where `tonlabs-cli` is built.
+Here we describe a contract deployment to the TON Blockchain Test Network (testnet) using `tonlabs-cli` tool. The following commands must be run from the directory where tonlabs-cli is located.
```

Comment:
I've heard that tonlabs-cli is renamed to tonos-cli (https://github.com/tonlabs/tonos-cli).
Should we continue to keep old name in docs?

Reply:
`tonlabs-cli` README https://github.com/tonlabs/TVM-linker/tree/master/tonlabs-cli still uses `tonlabs-cli` so I stick with it. When the tool is renamed, I update the docs.